src/chrome/content/rules/Cathay_Pacific.xml

```diff
@@ -0,0 +1,26 @@
+<ruleset name="Cathay Pacific">
+
+	<target host="cathaypacific.com" />
+	<target host="www.cathaypacific.com" />
+	<target host="cdn.cathaypacific.com" />
+	
+	<target host="cathaypacificcargo.com" />
+	<target host="www.cathaypacificcargo.com" />
+	
+	<target host="cxholidays.com" />
+	<target host="www.cxholidays.com" />
+	
+	<target host="dragonair.com" />
+	<target host="www.dragonair.com" />
+	
+	<rule from="^http://(?:www\.)?cathaypacific\.com/" to="https://www.cathaypacific.com/" />
```

Comment:
suggest to put a comment why lines 16 - 22 are necessary (timeout) for clarification. `(?:www\.)?` is not necessary since www.cathaypacific.com is covered by line 24.